### PR TITLE
ADD: Adding codeowners, issue template and pull template.

### PR DESCRIPTION
**Pull request by zssherman**
Adds codeowners file and an basic issue and pull request template.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+
+* @AdamTheisen
+* @zssherman
+* @mgrover1
```
Comment (mgrover1) [inline .github/CODEOWNERS:8]: Should we add Ken here too?

**Review by mgrover1 — CHANGES_REQUESTED** [review]
One minor suggestion/question, other than that, great! I like the more detailed PR template.

File: .github/CODEOWNERS (hunk 1 above)
Comment (zssherman) [reply .github/CODEOWNERS:8]: Yep good catch!

**Review by mgrover1 — APPROVED** [review]

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+* @kenkehoe
```
Comment (mgrover1) [inline .github/CODEOWNERS:9]:
```suggestion
* @kenkehoe
```

Comment (zssherman) [reply .github/CODEOWNERS:9]: I'm seeing the same change?

**Pull request merged**